inst/staticexports/package.R

```diff
@@ -71,11 +71,8 @@ system_file <- function(..., package = "base") {
   normalizePath(files, winslash = "/")
 }
 
-# A wrapper for `system.file()`, which caches the results, because
-# `system.file()` can be slow. Note that because of caching, if
-# `system_file_cached()` is called on a package that isn't installed, then the
-# package is installed, and then `system_file_cached()` is called again, it will
-# still return "".
+# A wrapper for `system.file()`, which caches the package file path.
+# Note, only non-empty paths are cached.
```

Comment:
```suggestion
# A wrapper for `system.file()`, which caches the package path because
# `system.file()` can be slow. If a package is not installed, the result won't
# be cached.
```